src/BatteryStateOfChargeService.cpp

```diff
@@ -1,13 +1,17 @@
 #include "BatteryStateOfChargeService.h"
+#include <stdio.h>
 
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
+    double ampHourTotal;
+    double ampHourUsed;
```

Comment:
Usually we reserve this area for constants, if you need a variable held between calls in your class, you should use a private member.